bridge/contracts/libraries/StakingNFT/StakingNFT.sol

```diff
@@ -498,6 +498,10 @@ abstract contract StakingNFT is
     function _mintNFT(address to_, uint256 amount_) internal returns (uint256 tokenID) {
         // this is to allow struct packing and is safe due to AToken having a
         // total distribution of 220M
+        require(
```

Comment:
I had to go confirm it, you can use &&, || in require statement && is logical and and || is logical or